homeassistant/components/hassio/ingress.py

```diff
@@ -185,7 +185,11 @@ def _init_header(request: web.Request, token: str) -> CIMultiDict | dict[str, st
 
     # Set X-Forwarded-For
     forward_for = request.headers.get(hdrs.X_FORWARDED_FOR)
-    connected_ip = ip_address(request.transport.get_extra_info("peername")[0])
+    if (peername := request.transport.get_extra_info("peername")) is None:
```

Comment:
Honestly, looking at this, I think this is incorrect (and also was incorrect as well).

This assumes the connected peer is the client, which doesn't have to be the case, this should already be processed by our middleware, and thus `request.remote` should contain the actual client IP.

Reply:
Oh nevermind 🤦 
It's a proxy and tries to pass as is. Morning!